test/ci/cpu_integration_test_linux.yml

```diff
@@ -47,21 +47,21 @@ jobs:
       cd /usr/share/miniconda/bin
       source activate interpret_cpu
       cd /home/vsts/work/1/s
-      pytest test/integration/test_notebook_msra_explainer.py --junitxml=./TEST--TEST.xml -o junit_suite_name="$(Agent.JobName)"
-    displayName: 'Run MSRA notebook and explainer'
+      pytest test/integration/test_notebook_unified_information_explainer.py --junitxml=./TEST--TEST.xml -o junit_suite_name="$(Agent.JobName)"
```

Comment:
it looks like it actually is called msra and not
test_notebook_msra_explainer.py
which is causing the tests to be cancelled

Reply:
:( Thanks for that!